src/internal.c

```diff
@@ -6024,6 +6223,16 @@ int InitSSL(WOLFSSL* ssl, WOLFSSL_CTX* ctx, int writeDup)
     ssl->tls13SecretCtx = NULL;
 #endif
 #endif
+#if defined(OPENSSL_EXTRA) && defined(HAVE_SECRET_CALLBACK)
```

Comment:
IsTLS() determines whether the current connection is TLS or not.

Adding callback setters is not correct here.  Do that in an initialize function.

Reply:
maybe, Github is showing you wrong function name. It is not implemented in IsTLS but InitSSL.